engines/hadesch/detection.cpp

```diff
@@ -0,0 +1,71 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * Copyright 2020 Google
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+#include "common/system.h"
+#include "common/savefile.h"
+
+#include "engines/advancedDetector.h"
+
+#include "hadesch/hadesch.h"
+
+static const PlainGameDescriptor hadeschGames[] = {
+	{"hadesch", "Hades Challenge"},
+	{0, 0}
+};
+
+static const char *const directoryGlobs[] = {
```

Comment:
It's a good idea to keep as much static data in your engine's namespace as possible. There's no reason why the meta engine can't refer to it like HadesCh::directoryGlobs. This will help avoid clashes with other engines in the future

Reply:
Done